WWW/smoorthi-dev/minidata_mutation_calling_v2.wdl

```diff
@@ -0,0 +1,376 @@
+version 1.0
+## Notes: Testing alignment for 3 samples
+## 
+## Samples:
+## MOLM13: Normal sample
+## CALU1: KRAS G12C mutant
+## HCC4006: EGFR Ex19 deletion mutant 
+##
+## Input requirements:
+## - combined fastq files for chromosome 12 and 7 +/- 200bp around the sites of mutation only
+##
+## Output Files:
+## - An aligned bam for all 3 samples with mutation calling
+## 
+## Workflow developed by Sitapriya Moorthi @ Fred Hutch LMD: 01/10/24 for use by DaSL @ Fred Hutch.
+## Workflow updated on : 02/13/24 by Sitapriya Moorthi
+
+struct referenceGenome {
+    File ref_fasta
+    File ref_fasta_index
+    File ref_dict
+    File ref_amb
+    File ref_ann
+    File ref_bwt
+    File ref_pac
+    File ref_sa
+    String ref_name
+}
+
+
+workflow minidata_mutation_calling_v2 {
+  input {
+    Array[File] allSamples
+    File normalFastq
```

Comment:
Beyond the test case, we need normalFastq to be also an array of Files. I think we should do this as if it is the real deal, and use our normal fastq twice.

Reply:
There was a particular reason I did not do that. The reason being that if we keep our normal separate we can write that when you run a WDL, it evaluates what inputs are needed for running the steps. So here for MuTect2 to run you will need the normals also to be aligned and calibrated.